kernel/kernel-impl/src/main/webapp/WEB-INF/db-components.xml

```diff
@@ -418,7 +418,7 @@
     <!-- Proxy which will keep only get the connection if one is actually needed,
        this is especially helpful when using hibernate or wrapping transactions around requests
        because it keeps the connection from being opened and held if it is not needed -->
-    <bean id="javax.sql.LazyDataSource" class="org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy">
+    <bean id="javax.sql.LazyDataSource" class="org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy" primary="true">
        <property name="targetDataSource"><ref bean="javax.sql.DataSource" /></property>
```

Comment:
Added to help Spring Boot find the datasource:

Parameter 0 of constructor in org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration required a single bean, but 5 were found:
	- helpDataSource: defined in file [E:\tomcats\tomcat-8.0.46\components\sakai-comp-help\WEB-INF\components.xml]
	- javax.sql.BaseDataSource: defined in file [E:\tomcats\tomcat-8.0.46\components\sakai-kernel-impl\WEB-INF\db-components.xml]
	- javax.sql.DataSource: defined in file [E:\tomcats\tomcat-8.0.46\components\sakai-kernel-impl\WEB-INF\db-components.xml]
	- javax.sql.LazyDataSource: defined in file [E:\tomcats\tomcat-8.0.46\components\sakai-kernel-impl\WEB-INF\db-components.xml]
	- javax.sql.LongDataSource: defined in file [E:\tomcats\tomcat-8.0.46\components\sakai-kernel-impl\WEB-INF\db-components.xml]
Action:
Consider marking one of the beans as @Primary, updating the consumer to accept multiple beans, or using @Qualifier to identify the bean that should be consumed